autoload/ctrlp/mrufiles.vim

```diff
@@ -56,16 +57,22 @@ fu! s:reformat(mrufs, ...)
 endf
 
 fu! s:record(bufnr)
-	if s:exclnomod && &l:modifiable | retu | en
-  if s:exclnomod && !&l:modifiable | retu | en
```

Comment:
These were for excluding files only opened for viewing, but not editing, such as (Vim) help files from the MRU.